### ISSUE TITLE
Frogger beyond Graphical Issues

### ISSUE THREAD

**Issue by Triticum0** [xemu-project/xemu#1209]
### Title

[Frogger Beyond](https://xemu.app/titles/4b4e0013#Frogger-Beyond)

### Bug Description

The games respawn and particle effects animation are coloured black when they should be white
![xemu-2022-07-26-20-27-12](https://user-images.githubusercontent.com/71127352/181096178-23bb3194-30f6-4f0a-a81d-19c8add8fb9f.png)
![xemu-2022-07-26-20-27-17](https://user-images.githubusercontent.com/71127352/181096189-838a73ae-3a18-45ec-bca8-3f4a03d1ce97.png)

### Expected Behavior

https://youtu.be/CdE61Nip6qs?t=200

### xemu Version

Version: 0.7.64
Branch: master
Commit: https://github.com/mborgerson/xemu/commit/248fc788c177e7b01ec1455b2005ee339332edbf
Date: Tue Jul 19 01:06:48 UTC 2022

### System Information

<html><body>

Field | Value
-- | --
OS | Windows 10
CPU | AMD Ryzen 5 2600 Six-Core Processor
Graphics Device | NVIDIA GeForce RTX 3060 Ti/PCIe/SSE2
Graphics Driver | 4.0.0 NVIDIA 512.95

</body>
</html>

### Additional Context

Here RenderDoc of the issues.

Spawn effect Renderdoc : https://drive.google.com/file/d/1AULof-Yac5qKd0ZiDDBLpJvMMx4YUcxv/view?usp=sharing

Question mark courrtion Renderdoc : https://drive.google.com/file/d/1-ooIxHzItLYjNy4DPGEakRv_gaW_4HXX/view?usp=sharing

Text name Renderdoc : https://drive.google.com/file/d/17LN2BGZixvdLSLoQDflHqEJ-JLgofZth/view?usp=sharing

**Comment by ghost** [comment]
yes, dark polygon glitches also cover a large part of the screen at times, which you haven't shown in the screenshots

**Comment by Triticum0** [comment]
@filevans Its been fixed
